packages/pluggableWidgets/maps-web/src/components/LeafletMap.tsx

```diff
@@ -22,9 +22,11 @@ export interface LeafletProps extends SharedProps {
  * If not, we reuse a leaflet icon that's the same as the default implementation should be.
  */
 const defaultMarkerIcon = new LeafletIcon({
-    iconRetinaUrl: require("leaflet/dist/images/marker-icon-2x.png"),
+    iconRetinaUrl: require("leaflet/dist/images/marker-icon.png"),
     iconUrl: require("leaflet/dist/images/marker-icon.png"),
-    shadowUrl: require("leaflet/dist/images/marker-shadow.png")
+    shadowUrl: require("leaflet/dist/images/marker-shadow.png"),
+    iconSize: [25, 41],
+    iconAnchor: [12, 41]
```

Comment:
These 2 are necessary to make the icon behave correctly when zooming in and out on the map according to https://github.com/PaulLeCam/react-leaflet/issues/306. basically, without these values the reference point of the icon would not be at the tip of the point marker. This means that when zooming in and out, the marker would appear to be on the wrong location.

Reply:
Is this also affect the OpenStreetMaps?

Reply:
yup, all maps except for google maps